ComfortZoneApp/app/src/main/java/com/example/comfortzone/flight/ui/FlightFragment.java

```diff
@@ -100,6 +111,11 @@ private void populateViews() {
         cityList.addAll(db.weatherDao().getAll());
         spSort.setAdapter(spinnerAdapter);
         tgCityDisplay.check(R.id.btnList);
+        if (isFahrenheit[0]) {
+            tvFahrenheit.setTypeface(Typeface.DEFAULT_BOLD);
+        } else {
+            tvCelsius.setTypeface(Typeface.DEFAULT_BOLD);
+        }
```

Comment:
you can even put this logic in the listener since it can also be reused in other surfaces, maybe just combine it with `degreeListeners()` and call them `initializeViews` or `init()`